src/tests/integration/test_generic_db_operations.py

```diff
@@ -265,15 +265,15 @@ def test_telemetry_execute_command_has_connection_info(
     "cell",
     [
         (
-            "%sqlplot histogram --with plot_something_subset --table\
+            "%sqlplot histogram --with plot_something_subset2 --table\
```

Comment:
is this name change required? if it's colliding with other tests, it's better to mock the SQLStore so they don't collide

Reply:
Reverted